src/AppLayout.tsx

```diff
@@ -10,26 +11,33 @@ export enum Widths {
 }
 
 function AppLayout() {
-    const [navigationOpen, setNavigationOpen] = useState(false);
-    const [navWidth, setNavigationWidth] = useState<Widths>(Widths.RAIL);
+    const [navigationConfig, setNavigationConfig] = useLocalStorage(
+        LocalStorageKeys.NAVIGATION_SETTINGS,
+        { open: true, width: 'FULL' }
```

Comment:
I am confused why we store width in any way. What is driving the decision for storing the width variable in local storage?

Reply:
If you would like the pane to default to being full on browser refresh, that is fine. Will remove.

Reply:
resolved in chat